### ISSUE TITLE
[Screen Readers-Ask HR Bot-Ask HR Virtual Assistant]: JAWS is narrating the suggestion invoked by the user twice upon invoking the Suggested Questionnaire button present in the “Ask HR Virtual Assistant” chat window.

### ISSUE THREAD

**Issue by KattaBhargav** [microsoft/BotFramework-WebChat#4295]
### PWD Impact:
Screen reader users are getting impacted if JAWS is narrating the suggestion invoked by the user twice upon invoking the Suggested Questionnaire button present in the “Ask HR Virtual Assistant” chat window.

### Test Environment:
OS: WIN10 1909 (OS Build 18363.836)
Browser: Chrome Version 83.0.4103.61 (Official Build) (64-bit)
URL: [https://microsoft.sharepoint.com/teams/HR_Web_Dev_Site/SitePages/AskHRBotTest.aspx](https://microsoft.sharepoint.com/teams/HR_Web_Dev_Site/SitePages/AskHRBotTest.aspx?xsdata=MDV8MDF8fDM4ZTRlMDY3ZGYwOTRmMmM0MWE2MDhkYTNjYmVhZTM4fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDB8MHw2Mzc4ODkwODY4OTA3MjE3MDJ8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxNVGs2TjJRd1lqVXpZell0TWpBNVlTMDBNVGhqTFdFM09XUXRNMlJqWVdOaU4yUTJORFUyWDJNd1lqazJZalZrTFROaE1EQXROR1l6WVMwNVpXRXhMVFZoTVdZell6RTNZVEZoWTBCMWJuRXVaMkpzTG5Od1lXTmxjdz09fHw%3D&sdata=R3dyTGJsN0xDWHZaSnhWaXRNK3l3UUlPa3lGNExuaGlvV2dLNlRkSkNvYz0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2Cv-kbhargav%40microsoft.com)
Screen Reader: JAWS 2019.1912.1

### Repro Steps:
Step 1: Enable JAWS and open the above URL in Chrome browser.
Step 2: Navigate to the “Ask HR Bot” Button in cursor mode and invoke it.
Step 3: Invoke any of the suggested questionnaire button present in the chat window.
Step 4: Verify the screen reader behavior upon invoking the suggestion questionnaire button present in the “Ask HR Virtual Assistant” chat window.

### Actual Result:
JAWS is narrating the suggestion invoked by the user twice upon invoking the Suggested Questionnaire button present in the “Ask HR Virtual Assistant” chat window.
Ex: JAWS is narrating as:  "User Said Ask a Generic HR Question sent at 
29th May 2020 2.16 P.M User Said Ask a Generic HR Question sent at 
29th May 2020 2.16 P.M. "

### Expected Result:
JAWS should narrate  the suggestion invoked by the user once upon invoking the Suggested Questionnaire button present in the “Ask HR Virtual Assistant” chat window.
JAWS should narrate as : "User Said Ask a Generic HR Question sent at 
29th May 2020 2.16 P.M."

### MAS Reference:
[MAS 4.2.1 - Object Information (33A)](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7Ba12f3779-bdf1-40c2-bbcc-0e314ea94858%7D&xsdata=MDV8MDF8fDI5ZjRiNTBmNzBmNTQ5NWU0MWE2MDhkYTNjYmVhZTM4fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDB8MHw2Mzc4ODkwODY4OTA3MjE3MDJ8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxNVGs2TjJRd1lqVXpZell0TWpBNVlTMDBNVGhqTFdFM09XUXRNMlJqWVdOaU4yUTJORFUyWDJNd1lqazJZalZrTFROaE1EQXROR1l6WVMwNVpXRXhMVFZoTVdZell6RTNZVEZoWTBCMWJuRXVaMkpzTG5Od1lXTmxjdz09fHw%3D&sdata=Q0ZjYTdNeS9EQnZHN21JTGNCaG5VVyt6WFFiUisvQ0tHcjl1TDlqZWcrST0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2Cv-kbhargav%40microsoft.com)

**Comment by KattaBhargav** [comment]
#A11yMAS;#A11ySev4;#HCL;#WCAG4.2.1;#Accessibility;#Win10-Edge(Chromium);#BotFrameworkWebChat-Aug21;#Benchmark;#CSEO:BUG5843513;#Firstparty:CSEO;

**Comment by KattaBhargav** [comment]
Bug copied from CSEO team Bug id:-[BUG 5843513](https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/5843513)

**Comment by axelsrz** [comment]
assigning @compulim

**Comment by compulim** [comment]
Action items:

1. When a message with suggested actions arrives, we should narrate it in live region twin
2. Suggested actions container should be a [`toolbar` role](https://www.w3.org/WAI/ARIA/apg/patterns/toolbar/)
3. (Optional and need asks) Carousel flipper buttons could have <kbd>TAB</kbd> disabled (`tabindex="-1"`) to reduce confusion and complexity when pressing <kbd>TAB</kbd> key to access the suggested action buttons

**Comment by compulim** [comment]
Reopen to pend for verification.

**Comment by compulim** [comment]
This is done in PR #4323. Testers did not verify it on time, so we assume this is fixed.

**Comment by v-prahu** [comment]
Updating the comments for Labels again to pull this bug in Kusto.

#A11yMAS;#A11ySev4;#HCL;#WCAG4.2.1;#Accessibility;#Win10-Edge(Chromium);#BotFrameworkWebChat-Aug21;#Linked:Bug5843513;#LinkedBug;
